.github/workflows/test.yml

```diff
@@ -5,10 +5,6 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
-    - uses: subosito/flutter-action@v2
```

Comment:
Should we also change this workflow to run on PR changes and merges to master? 
I guess now it only runs when you trigger it manually?

Reply:
Since Emran opened a PR to add a pipeline script, I'd close this PR and work on including the script into our actions

Reply:
Then we have a PR for a "final" pipeline action that has all requirements we want